### PR TITLE
plugin(apm-data): amend logs-apm component template

### PR DESCRIPTION
**Pull request by inge4pres**
Restore standard mode after LogsDB took precedence. Related: #114501

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)? ✅ 
- Have you followed the [contributor guidelines](https://github.com/elastic/elasticsearch/blob/main/CONTRIBUTING.md)? ✅ 
- If submitting code, have you built your formula locally prior to submission with `gradle check`? 🚫 
- If submitting code, is your pull request against main? Unless there is a good reason otherwise, we prefer pull requests against main and will backport as needed. ✅ 
- If submitting code, have you checked that your submission is for an [OS and architecture that we support](https://www.elastic.co/support/matrix#show_os)? ✅

### CODE REVIEW

File: x-pack/plugin/apm-data/src/main/resources/component-templates/logs-apm@settings.yaml [hunk 1]
```diff
@@ -6,4 +6,4 @@ _meta:
 template:
   settings:
     codec: best_compression
```
Comment (martijnvg) [inline x-pack/plugin/apm-data/src/main/resources/component-templates/logs-apm@settings.yaml:8]: I would also update the codec setting here to `index.codec`.

Comment (inge4pres) [reply x-pack/plugin/apm-data/src/main/resources/component-templates/logs-apm@settings.yaml:8]: Ah thanks 😄

Comment (inge4pres) [reply x-pack/plugin/apm-data/src/main/resources/component-templates/logs-apm@settings.yaml:8]: Is it worth creating the nesting then?

**Review by lahsivjar — APPROVED** [review]
🎉

**Review by salvatore-campagna — APPROVED** [review]

**Comment by inge4pres** [comment]
What's the right labels sorcery here?

**Comment by lahsivjar** [comment]
@inge4pres Do we need to backport this? Seems to only affect serverless so not needed I think but maybe we should backport.

**Comment by inge4pres** [comment]
> Do we need to backport this?

Seems like no, as long as #114501 is not going to land in 8.x too.

**Review by 1pkg — APPROVED** [review]

**Pull request merged**